### ISSUE TITLE
Some channels are not scheduled

### ISSUE THREAD

**Issue by Cycor** [meeb/tubesync#43]
At the moment I am monitoring 9 channels, but under Tasks I only see 3 index media tasks scheduled, all the other channel folder stay empty.
I got this a few times and used Reset Tasks to get it start downloading but it takes a very long time to reindex all channels.

Changing the channel settings seems to work but only once, the channels are downloaded but not added as an index media task.

Also, when a download failed, should it be added to the queue forever or does it give up after a few attempts, because I got a lot of failed downloads that aren't retried, I can make a new issue with those errors if you like.

I'm using the :latest tag because otherwise 'Reset Tasks' causes the 429 rate limit.

**Comment by meeb** [comment]
What image version are you running where you experience this? v0.8 or latest?

**Comment by Cycor** [comment]
I'm using the :latest tag because otherwise 'Reset Tasks' causes the 429 rate limit.

**Comment by meeb** [comment]
OK, thanks. There is work being done already on improving the scheduling tasks issue in other tickets so your issue should be resolved in the next release.

**Comment by meeb** [comment]
In more recent releases has this issue improved for you?

**Comment by Cycor** [comment]
Sorry for the late answer, yes, this problem has been solved

**Comment by meeb** [comment]
No problem, thanks for the feedback.
